File: Go/GORM/models/Position.go
package models

type Position uint

const (
	Boss Position = iota
	Manager
	Staffer
)

func (p Position) String() string {
	return map[Position]string{
		Boss:    "Boss",
		Manager: "Manager",
		Staffer: "Staffer",
	}[p]
}

File: JavaScript/TypeORM/src/entity/Position.js
export const Position = {
    Boss: 0,
    Manager: 1,
    Staffer: 2,

    /**
     *
     * @param {number} id
     * @returns string
     */
    toString(id) {
        return [
            "Boss",
            "Manager",
            "Staffer",
        ][id]
    }
}
